docs/schema-loader.md

```diff
@@ -1,50 +1,83 @@
 
-There are two ways to specify general CLI options in Schema Loader:
-  - Pass a ScalarDB configuration file and database/storage-specific options additionally.
-  - Pass the options without a ScalarDB configuration (Deprecated).
+Since managing the schema mapping and metadata for transactions can be difficult, you can use ScalarDB Schema Loader, which is a tool to create schemas that doesn't require you to need in-depth knowledge about schema mapping or metadata.
 
-Note that this tool supports only basic options to create/delete/repair/alter a table. If you want
-to use the advanced features of a database, please alter your tables with a database specific tool after creating them with this tool.
+You have two options to specify general CLI options in Schema Loader:
 
-# Usage
+- Pass the ScalarDB properties file and database-specific or storage-specific options.
+- Pass database-specific or storage-specific options without the ScalarDB properties file. (Deprecated)
 
-## Install
+{% capture notice--info %}
+**Note**
 
-The release versions of `schema-loader` can be downloaded from [releases](https://github.com/scalar-labs/scalardb/releases) page of ScalarDB.
+This tool supports only basic options to create, delete, repair, or alter a table. If you want to use the advanced features of a database, you must alter your tables with a database-specific tool after creating the tables with this tool.
+{% endcapture %}
 
-## Build
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+## Set up Schema Loader
+
+Select your preferred method to set up Schema Loader, and follow the instructions.
+
+<div id="tabset-1">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'Install', 'tabset-1')" id="defaultOpen-1">Install</button>
+  <button class="tablinks" onclick="openTab(event, 'Build', 'tabset-1')">Build</button>
+  <button class="tablinks" onclick="openTab(event, 'Docker', 'tabset-1')">Docker</button>
+</div>
+
+<div id="Install" class="tabcontent" markdown="1">
+
+You can download the release versions of Schema Loader from the [ScalarDB Releases](https://github.com/scalar-labs/scalardb/releases) page.
+</div>
+<div id="Build" class="tabcontent" markdown="1">
+
+If you want to build Schema Loader from the source, run the following command:
 
-In case you want to build `schema-loader` from the source:
 ```console
 $ ./gradlew schema-loader:shadowJar
 ```
-- The built fat jar file is `schema-loader/build/libs/scalardb-schema-loader-<version>.jar`
 
-## Docker
+Running that command builds a fat JAR file named `scalardb-schema-loader-<VERSION>.jar` in the `schema-loader/build/libs` folder, with `<VERSION>` matching the version of ScalarDB that you are using.
+</div>
+<div id="Docker" class="tabcontent" markdown="1">
```

Comment:
IMO, the tabs should be in order of recommendation. From that perspective, I guess the Docker way is the most recommended since it's the easiest to start using ScalarDB. What do you think?

Reply:
Understood. I think this will depend on which methods we decide to keep, according to the discussion in [this comment](https://github.com/scalar-labs/scalardb/pull/1196/files#r1371064322).

Considering that, I will mark this comment as resolved for now.